compiler/bindings/c/iree/compiler/dialects/iree_gpu.h

```diff
@@ -160,8 +167,20 @@ ireeHALExecutableTargetAttrGetGPUTargetInfo(MlirAttribute attr);
 MLIR_CAPI_EXPORTED ireeGPUTargetInfo ireeGPUTargetInfoGet(
     MlirContext mlirCtx, const char *arch, const int32_t *subgroupChoices,
     size_t numSubgroupChoices, const int32_t *workgroupSizes,
-    size_t numWorkgroupSizes, int64_t threadCount, int64_t memoryBytes,
-    const int32_t *mmaIntrinsics, size_t numMmaIntrinsics);
+    size_t numWorkgroupSizes, int32_t threadCount, int32_t memoryBytes,
+    const mma_intrinsic_t *mmaIntrinsics, size_t numMmaIntrinsics);
+
+struct ireeGPUMMAIntrinsicResult {
+  mma_intrinsic_t *mmaIntrinsicVals;
+  bool *isVirtual; // true if VirtualMMAIntrinsic, false if MMAIntrinsic.
```

Comment:
Could you make this a plural so that it's clear there is more than one value behind this pointer?